<?xml version='1.0' encoding='UTF-8'?>
<rss xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/" version="2.0">
  <channel>
    <title>CVE Feed for ovirt-engine-sdk-python -- ovirt-engine-sdk-python</title>
    <link>https://raw.githubusercontent.com/deepseas/cvelistV5monitor/main/feeds/ovirt-engine-sdk-python/ovirt-engine-sdk-python.rss</link>
    <description>The latest CVEs for ovirt-engine-sdk-python -- ovirt-engine-sdk-python</description>
    <docs>http://www.rssboard.org/rss-specification</docs>
    <generator>python-feedgen</generator>
    <lastBuildDate>Tue, 06 Aug 2024 11:20:26 +0000</lastBuildDate>
    <ttl>60</ttl>
    <item>
      <title>CVE-2014-0161|2024-08-06T09:05:38.992Z -- ovirt-engine-sdk-python -- ovirt-engine-sdk-python
</title>
      <link>https://www.cve.org/CVERecord?id=CVE-2014-0161</link>
      <description>ovirt-engine-sdk-python before 3.4.0.7 and 3.5.0.4 does not verify that the hostname of the remote endpoint matches the Common Name (CN) or subjectAltName as specified by its x.509 certificate in a TLS/SSL session. This could allow man-in-the-middle attackers to spoof remote endpoints via an arbitrary valid certificate.</description>
      <guid isPermaLink="false">CVE-2014-0161|2024-08-06T09:05:38.992Z</guid>
      <pubDate>Thu, 02 Jan 2020 17:30:53 +0000</pubDate>
    </item>
  </channel>
</rss>
